apps/demos/Demos/HtmlEditor/MarkdownSupport/description.md

```diff
@@ -1,4 +1,4 @@
-Our HtmlEditor outputs the markup in HTML format.
+DevExtreme HTML Editor outputs the markup in HTML format. To convert the output to Markdown, use the [unified](https://github.com/unifiedjs/unified) plugin library.
 
-The HtmlEditor stores the markup in the [value](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#value) property. In this demo, the value is displayed under the HtmlEditor.
+HTML Editor includes a [converter](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/) option. Define the converter with [toHtml](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/#toHtml) and [fromHtml](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/#fromHtml) functions and assign it to the converter property, as shown in this demo.
```

Comment:
I'm trying to minimize repetition, and also finalizing the description with something users can do in this demo. 
 
```suggestion
The code sets the the editor's [converter](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/) property to an object that implements two functions: [toHtml](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/#toHtml) and [fromHtml](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/converter/#fromHtml). 

Try to edit or format text in the HTML Editor control and see how the Markdown Preview section reflects those changes. 
```